lib/db/mysql/index.js

```diff
@@ -87,6 +88,9 @@ function checkDbPatchLevel(patcher) {
 }
 
 MysqlStore.connect = function mysqlConnect(options) {
+  if (options.logger) {
```

Comment:
It'd be more robust, but we didn't check before this diff, and the `options` object is freely used in the lines below.